Spigot-Server-Patches/0736-Add-command-line-option-to-load-extra-plugin-jars-no.patch

```diff
@@ -0,0 +1,64 @@
+From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
+From: jmp <jasonpenilla2@me.com>
+Date: Tue, 18 May 2021 14:39:44 -0700
+Subject: [PATCH] Add command line option to load extra plugin jars not in the
+ plugins folder
+
+ex: java -jar paperclip.jar nogui -add-plugin=/path/to/plugin.jar -add-plugin=/path/to/another/plugin_jar.jar
+
+diff --git a/src/main/java/org/bukkit/craftbukkit/CraftServer.java b/src/main/java/org/bukkit/craftbukkit/CraftServer.java
+index 033a0f1852afd6c1fd136ef07b611a9132adfa0c..79f81b14a513d1e2f25d2b212c2427d8f9ebc139 100644
+--- a/src/main/java/org/bukkit/craftbukkit/CraftServer.java
++++ b/src/main/java/org/bukkit/craftbukkit/CraftServer.java
+@@ -385,8 +385,13 @@ public final class CraftServer implements Server {
+ 
+         File pluginFolder = (File) console.options.valueOf("plugins");
+ 
+-        if (pluginFolder.exists()) {
+-            Plugin[] plugins = pluginManager.loadPlugins(pluginFolder);
++        // Paper start
++        if (true || pluginFolder.exists()) {
++            if (!pluginFolder.exists()) {
++                pluginFolder.mkdir();
```

Comment:
mkdirs: make parents too

Reply:
The else branch of the if I short circuit here was calling `mkdir`, however I can change it anyways